vcx/wrappers/java/src/main/java/com/evernym/sdk/vcx/credential/CredentialApi.java

```diff
@@ -116,6 +159,15 @@ public void callback(int command_handle, int err, String stringData) {
         }
     };
 
+    /**
+     * Takes the credential object and returns a json string of all its attributes
+     *
+     * @param credentailHandle Credential handle that was provided during creation. Used to identify credential object.
+     * @return completable future
+     * @throws VcxException Thrown if an error occurs when calling the underlying SDK.
+     * @see "Refer credentialSendRequest example for credential demo"
```

Comment:
Trivial grammar fix: "Refer" --> "Refer to"
Please fix this in all other places